### PR TITLE
Change background color to white

### PR DESCRIPTION
**Pull request by dishanest**
## Overview

Changed background color to white. 

## Changes Made

While the designs specify a very light gray for the background of Volume, it's SwiftUI does not yet support dynamically changing the color of the status bar. This results in a color discrepancy between the app background and the status bar (see attached screenshot). Otherwise the background is imperceptibly white. 

## Screenshots

<details>
  <summary> Before </summary>

IMG_1454

</details>

<details>
  <summary> After </summary>

IMG_1456

</details>

### CODE REVIEW

File: Volume/Supporting Views/WebView.swift [hunk 1]
```diff
@@ -17,6 +17,7 @@ struct WebView: UIViewRepresentable {
 
     func makeUIView(context: Context) -> WKWebView {
         let webView = WKWebView()
+        webView.backgroundColor = UIColor(Color.white)
```
Comment (Npikielny) [inline Volume/Supporting Views/WebView.swift:20]: Is this different than just `.white`?

File: Volume/Views/Onboarding/PublicationDetail/PublicationDetail.swift [hunk 2]
```diff
@@ -112,16 +112,14 @@ struct PublicationDetail: View {
...
                 PublicationDetailHeader(navigationSource: navigationSource, publication: publication)
                     .padding(.bottom)
-
                 Divider()
                     .background(Color.volume.buttonGray)
                     .frame(width: 100)
-
                 Header("Articles")
                     .padding()
+
```
Comment (Npikielny) [inline Volume/Views/Onboarding/PublicationDetail/PublicationDetail.swift:122]: I have absolutely no clue what the new line conventions in this project are

**Review by Npikielny — APPROVED** [review]
That's enough slices 🥲

File: Volume/Views/Onboarding/PublicationDetail/PublicationDetail.swift (hunk 2 above)
Comment (dishanest) [reply Volume/Views/Onboarding/PublicationDetail/PublicationDetail.swift:122]: Generally `if` and `switch` blocks need newlines around them, and UI components should have no newlines surrounding them unless it's a `Spacer`. That's what I've noticed and been following that.

**Review by amyqxhuang — APPROVED** [review]
looks good!! 🤩

**Pull request merged**